facebook/__init__.py

```diff
@@ -122,6 +122,29 @@ def get_objects(self, ids, **args):
         args["ids"] = ",".join(ids)
         return self.request(self.version + "/", args)
 
+    def search_objects_by(self, type, **args):
+        """Fetches all objects of a given type from the graph.
+
+        Valid types are: event, group, page, place, placetopic, and user
+        Most types require the argument q, except:
+        - place requires q, addres or center
+        - placetopic doesn't require any additional argument
+
+        https://developers.facebook.com/docs/graph-api/using-graph-api
```

Comment:
https://developers.facebook.com/docs/graph-api/using-graph-api#search would be a better link to have here.